modules/llrt_stream/src/readable.rs

```diff
@@ -223,14 +225,16 @@ where
         readable: T,
         on_end: C,
     ) -> Result<Receiver<bool>> {
-        Self::do_process(this, ctx, readable, on_end)
+        Self::do_process(this, ctx, readable, on_end, None, None)
     }
 
     fn do_process<T: AsyncRead + 'js + Unpin, C: FnOnce() + Sized + 'js>(
         this: Class<'js, Self>,
         ctx: &Ctx<'js>,
         readable: T,
         on_end: C,
+        combined_std_buffer: Option<Arc<Mutex<Vec<u8>>>>,
+        cb: Option<Function<'js>>,
```

Comment:
Is there a case where we would want to use a combined_std_buffer independently from cb? All usages in this PR assume that cb and combined_std_buffer are either some or none.
I am not really a want of adding it to the stream, I wonder if we could instead either pipe the stdout/err through an accumulation middleware before forwarding it to the DefaultReadableStream.